### ISSUE TITLE
Smart Filter/Playlist

### ISSUE THREAD

**Issue by kascote** [clangen/musikcube#140]
will be cool to have some kind of advance filter and/or smart playlist.
for example filter like this: 

get only one genre and one artist
`genres:classical artists:bach`

get all stuff except something
`genres:!classical`

I saw on the API that there are some kind of support for this already (play_tracks_by_category, query_tracks). But dont saw an end point for create new playlists from the api.

**Comment by clangen** [comment]
Yeah, this is something musikcube will eventually support. I'm still trying to figure out how exactly I want to do this, but I want to make sure it lends itself well to creating a nice UI around it. It will probably be a little query language that has support for a small set of matching and logical operators.

**Comment by vdawg-git** [comment]
Small bumb, as I can't find any app except of Rhythmbox supporting this.

And Rhythmbox is not that good at it.
